imap/http_jmap.c

```diff
@@ -914,6 +916,9 @@ static int _create_upload_collection(const char *accountid,
         cyrus_acl_set(&newacl, httpd_userid, ACL_MODE_SET,
                       JACL_READITEMS | JACL_WRITE, NULL, NULL);
 
+        xsyslog(LOG_ERR, "reset ACL", "userid=<%s> newacl=<%s>",
```

Comment:
is LOG_ERR the right thing to use here or should this be LOG_NOTICE

Reply:
LOG_NOTICE indeed is the better choice